debian/base/Dockerfile

```diff
@@ -37,11 +40,12 @@ ENV SCALA_VERSION="2.11"
 ENV CONFLUENT_MAJOR_VERSION="3"
 ENV CONFLUENT_MINOR_VERSION="3"
 
-ENV CONFLUENT_PATCH_VERSION="0"
+ENV CONFLUENT_PATCH_VERSION="0~SNAPSHOT"
 ENV CONFLUENT_VERSION="$CONFLUENT_MAJOR_VERSION.$CONFLUENT_MINOR_VERSION.$CONFLUENT_PATCH_VERSION"
 ENV CONFLUENT_DEB_VERSION="1"
 
-ENV KAFKA_VERSION="0.11.0.0-SNAPSHOT"
+# Add ~SNAPSHOT (with tilde) for internal releases
+ENV KAFKA_VERSION="0.11.0.0~SNAPSHOT"
```

Comment:
After 3.3.0 is released, we will need to manually update all the files with different repo urls and also remove snapshot versions from here. If we can wrap everything in scripts to automatically change based on the git branch to produce release vs snapshots, it would be better. 

We have too many manual stuff that we have to do for each release / version, I think we need to start prioritizing on writing automated scripts as we build features rather than waiting until later. So I would recommend doing that now if possible.